### PR TITLE
chore: improve bug issue template

### PR DESCRIPTION
**Pull request by rarkins**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — DISMISSED** [review]

**Comment by rarkins** [comment]
Confusing thing is that it passes validation locally for me:

```
❯ git status
On branch chore/21034-issue-templates
Your branch is up to date with 'origin/chore/21034-issue-templates'.

nothing to commit, working tree clean

~/src/github/renovatebot/renovate chore/21034-issue-templates
❯ yarn prettier && echo "prettier done"
yarn run v1.22.15
$ prettier --cache --check "**/*.{ts,js,mjs,json,md,yml}"
Checking formatting...
All matched files use Prettier code style!
✨  Done in 3.18s.
prettier done
```

**Review by HonkingGoose — COMMENTED** [review]
I'm happy with the changes, but the linter is complaining.

**Comment by viceice** [comment]
> Confusing thing is that it passes validation locally for me:
> 
> ```
> ❯ git status
> On branch chore/21034-issue-templates
> Your branch is up to date with 'origin/chore/21034-issue-templates'.
> 
> nothing to commit, working tree clean
> 
> ~/src/github/renovatebot/renovate chore/21034-issue-templates
> ❯ yarn prettier && echo "prettier done"
> yarn run v1.22.15
> $ prettier --cache --check "**/*.{ts,js,mjs,json,md,yml}"
> Checking formatting...
> All matched files use Prettier code style!
> ✨  Done in 3.18s.
> prettier done
> ```

probably older node modules locally?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -44,30 +45,14 @@ body:
...
-          - GitLab (.com and CE/EE)
-    validations:
-      required: false
-
   - type: dropdown
     id: regression-error
     attributes:
       label: Was this something which used to work for you, and then stopped?
       options:
+        - '-- Please select from the below --'
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.yml:53]: I'm seeing this error message when I view this file on GitHub:

> There is a problem with this template
>
> YAML syntax error: (): did not find expected '-' indicator while parsing a block collection at line 53 column 9. [Learn more about this error.](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms)

You probably need to drop the `--` characters to get the form to work properly.

Link to the warning: https://github.com/renovatebot/renovate/blob/8bde1049b47afb6ebbee410105e909f3bf2fa5cf/.github/ISSUE_TEMPLATE/bug_report.yml

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -14,8 +14,9 @@ body:
     attributes:
       label: How are you running Renovate?
       options:
+        - '-- Please select from the below --'
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.yml:17]:
```suggestion
        - 'Placeholder value, please select the correct response from the dropdown'
```

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (HonkingGoose) [reply .github/ISSUE_TEMPLATE/bug_report.yml:53]:
```suggestion
        - 'Placeholder value, please select the correct response from the dropdown'
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -107,7 +92,8 @@ body:
       label: Have you created a minimal reproduction repository?
       description: Please read the [minimal reproductions documentation](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to learn how to make a good minimal reproduction repository.
       options:
-        - 'I have read the minimal reproductions documentation and linked to such a repository in the bug description'
+        - '-- Please select from the below --'
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.yml:95]:
```suggestion
        - 'Placeholder value, please select the correct response from the dropdown'
```

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
I hope this fixes the linter complaints. 😄

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -44,30 +45,14 @@ body:
...
-    validations:
-      required: false
-
   - type: dropdown
     id: regression-error
     attributes:
       label: Was this something which used to work for you, and then stopped?
       options:
+        - 'Placeholder value, please select the correct response from the dropdown'
+        - 'I\'m trying to get this working for the first time'
```
Comment (HonkingGoose) [inline .github/ISSUE_TEMPLATE/bug_report.yml:54]: I'm still get the warning message when looking at the "fixed" file on GitHub. 😞 

Maybe the `'I\'m` part is causing problems?

https://github.com/renovatebot/renovate/blob/chore/21034-issue-templates/.github/ISSUE_TEMPLATE/bug_report.yml

Comment (rarkins) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Seems like it's fixed now

**Review by HonkingGoose — APPROVED** [review]
The form preview is working properly now. 🥳

**Pull request merged**